src/chi-vue/src/components/data-table/DataTableTooltip.tsx

```diff
@@ -1,34 +1,58 @@
 import { Component, Vue, Prop } from 'vue-property-decorator';
 import Tooltip from '@/components/tooltip/tooltip';
-import { UTILITY_CLASSES } from '@/constants/classes';
 import { TOOLTIP_ANIMATION_DELAY } from '@/constants/constants';
+import { UTILITY_CLASSES } from '@/constants/classes';
 
 @Component({})
 export default class DataTableTooltip extends Vue {
   @Prop() msg!: string;
+  @Prop({ default: false }) header?: boolean;
```

Comment:
I believe header defaults to false without needing `{ default: false }` so it could be removed

Reply:
Good catch, done!